src/chart/mesh.rs

```diff
@@ -308,6 +312,29 @@ where
         self.axis_style = Some(style.into());
         self
     }
+
+    /// Set the maximum number of divisions for the fine grind grid
```

Comment:
I'm not a native speaker, but I've never heard of a "grind grid". Maybe this can be described better.